rdtools/test/filtering_test.py

```diff
@@ -333,3 +334,9 @@ def test_normalized_filter_default():
     pd.testing.assert_series_equal(normalized_filter(
                         pd.Series([0.01 - eps, 0.01 + eps, 1e308])),
                         pd.Series([False, True, True]))
+
+
+def test_hampel_filter():
+    a = pd.Series([5, 4, 7, 3, 4, 5, 4, 100, 3, 4, 5, 3, 5, 3, 5, 4, 12])
```

Comment:
Should we have tests with some NaN values as well?

Reply:
done